examples/mobile/HomeApp/webclip/tv-plus-channels/css/style.css

```diff
@@ -0,0 +1,4 @@
+.app-icon {
```

Comment:
Should we  define it as css separately?
Is not include the Card elements?

Reply:
> Would you also add the container, please?
> ![container](https://user-images.githubusercontent.com/31196177/100301720-18c7f400-2fdc-11eb-9002-09d0b0285786.png)

I'll change this.

Reply:
We have to define somewhere the icon image `../images/title/tvplus.png`.
You don't want this separate file. Do you prefer url to icony to be defined in webclip.html?

Reply:
How about your think?
I think it's better to be defined in webclip.html by <img src>.
In case of list icon, we also add in the html.

					<div class="ui-li-icon">
						<img src="images/tw_list_icon_sound.svg"/>
					</div>

Reply:
It is ok for me. I'll change this.